src/o365/teams/commands/team/team-add.ts

```diff
@@ -54,15 +131,66 @@ class TeamsTeamAddCommand extends GraphCommand {
       });
   }
 
+  private waitUntilFinished(operationUrl: string, resolve: () => void, reject: (error: any) => void, cmd: CommandInstance, dots?: string, timeout?: NodeJS.Timer): void {
+    if (!this.debug && this.verbose) {
+      dots += '.';
+      process.stdout.write(`\r${dots}`);
+    }
+
+    const requestOptions: any = {
+      url: `${this.resource}/beta${operationUrl}`,
+      headers: {
+        accept: 'application/json;odata.metadata=minimal'
+      },
+      json: true
+    };
+
+    request
+      .get(requestOptions)
+      .then((res: any) => {
+        const response = res as TeamsAsyncOperation;
```

Comment:
We can avoid the cast by changing the lines above to:

```ts
      .get<TeamsAsyncOperation>(requestOptions)
      .then((response: TeamsAsyncOperation) => {
```

Reply:
Nice tip 👍🏻